intel_extension_for_transformers/llm/runtime/graph/models/whisper/CMakeLists.txt

```diff
@@ -0,0 +1,19 @@
+#  Copyright (c) 2023 Intel Corporation
+#
+#  Licensed under the Apache License, Version 2.0 (the "License");
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+#  limitations under the License.
+
+set(TARGET whisper)
+add_library_w_warning(${TARGET} whisper.cpp whisper_utils.cpp)  # no (gpt) model utils needed
+target_compile_features(${TARGET} PUBLIC cxx_std_11) # don't bump
+set_target_properties(${TARGET} PROPERTIES POSITION_INDEPENDENT_CODE ON)
+target_link_libraries(${TARGET} PUBLIC ne_layers ${LLAMA_EXTRA_LIBS} jblas::jblas)
```

Comment:
I think current master already remove this

Reply:
https://github.com/intel/intel-extension-for-transformers/blob/e764bb57b45ec0b4bb3366ea29aa91e36273415e/intel_extension_for_transformers/llm/runtime/graph/models/llama/CMakeLists.txt#L19
we still need this cmake in main

Reply:
I mean ```${LLAMA_EXTRA_LIBS}```

Reply:
Yes, I got it.
I will build a PR, and add them into TO List